text/0000-localizing-api-messages.md

```diff
@@ -0,0 +1,135 @@
+
+- Start Date: 03-29-2022
+- RFC PR: 
+- FOLIO Issue: 
+- Submitter : Radhakrishnan Gopalakrishnan (rgopalakrishnan@ebsco.com)
+- Co-Submitter(s) : Zak Burke (zburke@ebsco.com)
+- Sub Group :
+
+# Localizing API (Backend) Messages
+
+## Summary
+  The purpose of this RFC is to define an approach that allows API developer to provide
+messages in the API response depending on the language preference set in the request. 
+
+
+## Motivation
+
+- Help users understand the information coming from the application backend using a language they understand better
+- To allow FOLIO platform to be used by users across the world 
+- To create a tremendous opportunity for growth that we can never could have achieved within just one country.
+
+### Terminology
+- Static Message - E.g. _This is something you need translate_
```

Comment:
> When going through JIRA, I found references to messages with line breaks, messages in HTML format etc.. I called these out mainly to highlight the type of messages this RFC handles. Messages in its most basic forms.

I appreciate the thorough analysis of the wide array of JIRA issues. I am a little concerned that this could lead to us combining all of those sources of information together in a way that doesn't really express the intended whole. 

Maybe these could be combined together - `static message with placeholder(s)` automatically covers static messages without placeholders.

And that statement could be moved to the section that describes the scope of this RFC.